db/warehouse/migrate/20240531020035_migrate_hmis_form_data.rb

```diff
@@ -0,0 +1,20 @@
+class MigrateHmisFormData < ActiveRecord::Migration[7.0]
+  def up
+    raise unless Hmis::Form::FormProcessor.column_names.include?('backup_values')
+    raise unless Hmis::Form::Definition.column_names.include?('backup_definition')
+
+    # Rake::Task.clear # clear previously loaded tasks
+    # Rails.application.load_tasks # reload tasks from the application
+
+    Rake::Task['driver:hmis:migrate_hmis_link_ids'].invoke
+  end
+
+  def down
+    raise 'refusing destructive rollback in production' if Rails.env.production?
```

Comment:
@ttoomey maybe we should wrap these in a `HmisEnforcement.hmis_enabled?` check so they skip altogether on other environments. What do you think ( cc @eanders ) ?

Reply:
@gigxz  I guess that should be fine to add an early-return so the migration runs but it's a no-op.

It probably doesn't matter though since it looks like non-hmis installations wouldn't even have form definition records to migrate; it seems like they aren't seeded unless `ENV['ENABLE_HMIS_API'] == 'true'`